recipes/sqlalchemy-jsonapi/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "SQLAlchemy-JSONAPI" %}
+{% set version = "4.0.8" %}
+{% set sha256 = "3f529ea8dd707760bdf4146862f102467b376ef9b618d97d3a8e222d12bb6dc6" %}
+
+package:
+  name: sqlalchemy-jsonapi
+  version: "{{ version }}"
+
+source:
+  fn: "{{ name }}-{{ version }}.tgz"
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Would recommend adding `-B` after `python` and before `setup.py`. This should avoid compiling dependencies and including them in the package.

xref: https://github.com/conda-forge/conda-forge.github.io/issues/264